### PR TITLE
Update # of tests, # cities, add Google Analytics

### PR DESCRIPTION
**Pull request by critzo**
This PR adds Google Analytics code.

### CODE REVIEW

File: src/containers/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -39,8 +39,8 @@ class HomePage extends PureComponent {
         <Helmet title="Home" />
         <div className="feature">
           <h1>Measurement Lab Visualizations</h1>
-          <p>Measurement Lab records and analyzes over <strong>200 million</strong> user generated
-             internet speed tests from over <strong>4,000 cities</strong>.
+          <p>Measurement Lab records and analyzes over <strong>38 million</strong> user generated
```
Comment (georgiamoon) [inline src/containers/HomePage/HomePage.jsx]: ok @pboothe said we were nearing 1 billion tests.. and this is lower than 200 million, so now I'm confused?

Comment (critzo) [reply src/containers/HomePage/HomePage.jsx]: I guess we need to decide how we're counting this. I queried for completed download tests.

**Comment by critzo** [comment]
@georgiamoon While we discuss the total number of tests and cities, I've modified the branch in this PR to just include the GA code. PTAL?

**Comment by fernandalavalle** [comment]
LGTM to me but I see you tagged @georgiamoon, so I'll let her give the final ok

**Pull request merged**